hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/DiskBalancerConfiguration.java

```diff
@@ -43,12 +43,12 @@ public final class DiskBalancerConfiguration {
           "utilization of the volume(used space to capacity ratio) differs" +
           " from the utilization of the datanode(used space to capacity ratio" +
           " of the entire datanode) no more than the threshold.")
-  private double threshold = 10d;
+  private String threshold = "10";
 
   @Config(key = "max.disk.throughputInMBPerSec", type = ConfigType.AUTO,
       defaultValue = "10", tags = {ConfigTag.DISKBALANCER},
       description = "The max balance speed.")
-  private double diskBandwidth = 10;
+  private long diskBandwidth = 10;
```

Comment:
Please check the type of all key, use the proper type instead of ConfigType.AUTO if possible.